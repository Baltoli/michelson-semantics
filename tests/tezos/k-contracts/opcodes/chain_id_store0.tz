$contract { parameter unit;
storage (option chain_id);
code { DROP; CHAIN_ID; SOME; NIL operation; PAIR }
 } ;
$param Unit ;
$storage None ;
$ExpectedOutput (Some 0x7a06a770) ;

