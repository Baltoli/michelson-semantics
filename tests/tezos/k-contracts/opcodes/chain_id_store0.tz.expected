<generatedTop>
  <k>
    #VerifyStorage ( Some #ChainId ( 2047256432 ) )
  </k>
  <stack>
    Pair .List Some #ChainId ( 0 )
  </stack>
  <paramtype>
    unit .AnnotationList
  </paramtype>
  <storagetype>
    option .AnnotationList chain_id .AnnotationList
  </storagetype>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( "InvalidMyAddr" )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( "InvalidSourceAddr" )
  </sourceaddr>
  <senderaddr>
    #Address ( "InvalidSenderAddr" )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
